index.rst

```diff
@@ -7,9 +7,13 @@ Welcome to Piwik PRO Developers documentation!
 =====================================
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Mobile SDK documentation
-   :name: test
-   :glob:
+   :maxdepth: 1
 
-   sdk/*
\ No newline at end of file
+   sdk/index
+
+
+
+.. toctree::
```

Comment:
You don't need separate `toctree` if you don't use captions and names. Just link all documents in one ToC.